test/metabase/util/queue_test.clj

```diff
@@ -127,6 +127,19 @@
 (defn- thread-name-running? [name]
   (some #(= name (.getName ^Thread %)) (keys (Thread/getAllStackTraces))))
 
+(defmacro ^:private await-test
```

Comment:
nit: maybe it's just me, but I read `(await-test foo? ...)` as "wait *until* foo", i.e. wait until the condition is true, whereas `(await-test-while foo? ...)` makes it clearer (to me anyway) that it will wait while the condition holds.